docs/Introduction.md

```diff
@@ -12,4 +12,4 @@ The aggregated collection of control class and datatype models is published in t
 
 Models of both classes and datatypes published either in the [Framework](Framework.md) or in one of the [Feature sets](Feature%20sets.md) are __standard__ models and MUST have their names prefixed with `"Nc"`.
 
-Device models can contain non-standard or [vendor-specific](Framework.md#ncclassid) classes by deriving standard classes. Vendor-specific models MUST NOT have their names prefixed with `"Nc"`.
+Device models can contain [non-standard](Framework.md#ncclassid) classes by deriving standard classes. The main use case is to allow vendors to expose vendor specific functionality in their device implementations. Non-standard models MUST NOT have their names prefixed with `"Nc"`.
```

Comment:
Model rather than expose?
```suggestion
Device models can contain [non-standard](Framework.md#ncclassid) classes by deriving standard classes. The main use case is to allow vendors to model vendor specific functionality in their device implementations. Non-standard models MUST NOT have their names prefixed with `"Nc"`.
```